website/content/en/docs/reference/configuration/sources/http_scrape.md

```diff
@@ -0,0 +1,14 @@
+---
+title: HTTP
+description: Collect observability data emitted by an [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Client_request) server
```

Comment:
(I assume this is copy-pasted from the existing `http` source) This probably needs to change both here and on the existing `http` source to provide better clarity that one is _pulling_ data from a remote target and one is _pushed_ data.

Reply:
Ah - @neuronull to be clear, not suggesting you should fix up the `http` source's description here, I just remember it being confusing to users before if it was a pull or push source.

Reply:
Addressed in https://github.com/vectordotdev/vector/pull/13793/commits/5f7c44231f801eb5ee0be2f2a3549efb6c908e8c